git_machete/docs.py

```diff
@@ -341,26 +341,25 @@
           <b>Options:</b>
             <b>--draft</b>    Creates the new PR as a draft.
 
-        <b>`retarget-pr`:</b>
-
-          Sets the base of the current branch's PR to upstream (parent) branch, as seen by git machete (see `git machete show up`).
-
-        <b>`fetch-prs <PR-number-1> ... <PR-number-N> [--all] [--mine] [--by]`:</b>
+        <b>`fetch-prs[ --all | --by=<github-login> | --mine | <PR-number-1> ... <PR-number-N>`:</b>
```

Comment:
Nit:
```suggestion
        <b>`fetch-prs [--all | --by=<github-login> | --mine | <PR-number-1> ... <PR-number-N>]`:</b>
```

Reply:
Pls also update .rst docs for https://git-machete.readthedocs.io/en/stable/ ! we still have two separate places to keep the same docs... at least until https://github.com/VirtusLab/git-machete/issues/211 is done